sphinx_gallery/gen_rst.py

```diff
@@ -512,11 +511,8 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     ref_fname = example_file.replace(os.path.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot') or not gallery_conf['plot_gallery']:
-        convert_func = dict(code=codestr2rst, text=text2string)
-        for blabel, bcontent in script_blocks:
-            example_rst += convert_func[blabel](bcontent) + '\n'
-    else:
+    filename_pattern = gallery_conf.get('filename_pattern')
```

Comment:
For me all the defaults are in the dictionary in gen_gallery.py, one place for everything and only there.
So no user is forced to change their conf.py if they don't feel to play around the `filename_pattern`, the plot default is there